google-http-client/src/test/java/com/google/api/client/util/ArrayMapTest.java

```diff
@@ -100,4 +103,52 @@ public void testSet() {
     } catch (IndexOutOfBoundsException e) {
     }
   }
+
+  public void testIteratorRemove1() {
+    ArrayMap<String, String> map = new ArrayMap<String, String>();
+    map.put("a", "a");
+    map.put("b", "b");
+    map.put("c", "c");
+    Iterator<Map.Entry<String,String>> iter = map.entrySet().iterator();
+    while (iter.hasNext()) {
+        Map.Entry<String,String> entry = iter.next();
+        if(!"all".equalsIgnoreCase(entry.getKey())){
+            iter.remove();
+        }
```

Comment:
Use consistent indentation with the current code (2-space)

Reply:
fixed. thanks!